js/src/views/Account/Shapeshift/OptionsStep/optionsStep.js

```diff
@@ -54,30 +54,26 @@ export default class OptionsStep extends Component {
       );
     }
 
+    coins = coins.map(this.renderCoinSelectItem);
+
     return (
       <div className={ styles.body }>
         <Form>
-          <Select
-            className={ styles.coinselector }
-            hint={
-              <FormattedMessage
-                id='shapeshift.optionsStep.typeSelect.hint'
-                defaultMessage='the type of crypto conversion to do'
-              />
-            }
-            label={
-              <FormattedMessage
-                id='shapeshift.optionsStep.typeSelect.label'
-                defaultMessage='fund account from'
-              />
-            }
+          <div style={ { display: 'block', paddingTop: '15px', paddingBottom: '5px', fontSize: '16px', pointerEvents: 'none', userSelect: 'none', color: 'rgba(0, 0, 0, 0.298039)' } }>
+            <FormattedMessage
+              id='shapeshift.optionsStep.typeSelect.label'
+              defaultMessage='Fund account from'
+            />
+          </div>
+          <Dropdown
+            placeholder='Choose a crypto-currency'
+            fluid
+            search
+            selection
+            options={ coins }
```

Comment:
As per the comment above, not convinced we need any of `fluid`, `search` & `selection` passed in for our current usage? (We can always expand, but try and keep the footprint of the UI API as small as possible)